compiler/src/java_plugin/cpp/java_generator.cpp

```diff
@@ -642,6 +642,32 @@ static void PrintBindServiceMethod(const ServiceDescriptor* service,
   p->Print("}\n");
 }
 
+static void PrintAbstractServiceClass(const ServiceDescriptor* service,
+                                   map<string, string>* vars,
+                                   Printer* p,
+                                   ProtoFlavor flavor) {
+  if (flavor != ProtoFlavor::NORMAL) {
```

Comment:
I just left this one up for PR debate and have no opinions. Should I just remove this restriction then?